packages/lit-dev-content/site/tutorials/tutorials.11tydata.js

```diff
@@ -39,6 +39,9 @@ module.exports = async () => {
 
     // Build
     loadTutorialData('brick-viewer'),
+
+    // svg templates and patterns
```

Comment:
for the tutorial plugin to work, please don't provide new comments other than the ones provided.

LOL I know it's jank AF but it's how I could get it to inject these statements into the file when you create a new tutorial

Reply:
done lol :)